lib/convection/control/stack.rb

```diff
@@ -85,6 +85,7 @@ def initialize(name, template, options = {})
         @resources = {}
         @current_template = {}
         @last_event_seen = nil
+        @retries = 0
```

Comment:
I'm not exactly sure if I needed to reset `@retries` within each method that was going to use this or if we get a new instance of this class each time.